### DISCUSSION TITLE
Can we upload music with title and uploader name ?

### DISCUSSION THREAD

**Discussion by mhamadsu777** [Nekmo/telegram-upload#123] [Ideas]
Can we upload music with title and uploader name and more ?

**Comment by tissole** [comment]
Yes, the script uses Hachoir which extracts metadata (artist name, song name, etc), and if you want to put the uploader's name you use --caption argument followed by the uploader's name.
https://docs.nekmo.org/telegram-upload/usage.html#telegram-upload
